src/index.ts

```diff
@@ -4,7 +4,9 @@ import * as dotenv from "dotenv";
 dotenv.config();
 
 mongoose
-  .connect(process.env.MONGODB_URI!)
+  .connect(
+    "mongodb+srv://farnick1148:o1vvfQtCYh4ehKJK@cluster0.d9r34ci.mongodb.net/banking-system/?retryWrites=true&w=majority"
```

Comment:
Please don't hardcode credentials

Reply:
okay, i had them in .env, but i was told to remove .env, so i hardcoded it for you to use it, since it's not real-life application.

Reply:
Yeah, it's not a real app but it's not a good practice to push credentials into remote repo